js/everything.js

```diff
@@ -474,7 +473,13 @@ $(function() {
             );
         }
     }
-
+    if ( /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) )
+    {
+        $("#container").ready(function() {
+          $("#container").html("<p></p>");
+          $("#change-mobile").html("Whoops! Sorry , we don't support mobile devices.");
+      });
+    }
```

Comment:
The last statement inside this `if` should be a `return;` we don't want to try doing anything else if we're on mobile.
We should also make sure the demo button is hidden at this point since it won't work